redpanda/templates/services.cluster.yaml

```diff
@@ -26,21 +26,21 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name | quote }}
     app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
     app.kubernetes.io/component: {{ template "redpanda.name" . }}
-  {{- with .Values.labels }}
+  {{- with .Values.commonLabels }}
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
   type: ClusterIP
   publishNotReadyAddresses: true
   sessionAffinity: None
   ports:
-      {{- if (first .Values.config.pandaproxy.pandaproxy_api).external.enabled }}
+      {{- if (first .Values.listeners.rest.endpoints).external.enabled }}
```

Comment:
I've changed pandaproxy in values.yaml to rest to be a more meaningful, descriptive name to the user.